x/auth/vesting/client/cli/tx.go

```diff
@@ -79,3 +86,167 @@ timestamp.`,
 
 	return cmd
 }
+
+type VestingData struct {
+	StartTime int64         `json:"start_time"`
+	Periods   []InputPeriod `json:"periods"`
+}
+
+type InputPeriod struct {
+	Coins  string `json:"coins"`
+	Length int64  `json:"length"`
+}
+
+// readScheduleFile reads the file at path and unmarshals it to get the schedule.
+// Returns start time, periods, and error.
+func readScheduleFile(path string) (int64, []types.Period, error) {
+	contents, err := ioutil.ReadFile(path)
+	if err != nil {
+		return 0, nil, err
+	}
+	var data VestingData
+	err = json.Unmarshal(contents, &data)
+	if err != nil {
+		return 0, nil, err
+	}
+	startTime := data.StartTime
+	var periods []types.Period
+	for i, p := range data.Periods {
+		amount, err := sdk.ParseCoinsNormalized(p.Coins)
+		if err != nil {
+			return 0, nil, err
+		}
+		if p.Length < 1 {
+			return 0, nil, fmt.Errorf("invalid period length of %d in period %d, length must be greater than 0", p.Length, i)
+		}
+		period := types.Period{Length: p.Length, Amount: amount}
+		periods = append(periods, period)
+	}
+	return startTime, periods, nil
+}
+
+// NewMsgCreateClawbackVestingAccountCmd returns a CLI command handler for creating a
+// MsgCreateClawbackVestingAccount transaction.
+func NewMsgCreateClawbackVestingAccountCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-clawback-vesting-account [to_address]",
+		Short: "Create a new vesting account funded with an allocation of tokens, subject to clawback.",
+		Long: `Must provide a lockup periods file (--lockup), a vesting periods file (--vesting), or both.
+If both files are given, they must describe schedules for the same total amount.
+If one file is omitted, it will default to a schedule that immediately unlocks or vests the entire amount.
+The described amount of coins will be transferred from the --from address to the vesting account.
+Unvested coins may be "clawed back" by the funder with the clawback command.
+Coins may not be transferred out of the account if they are locked or unvested, but may be staked.
+Staking rewards are subject to a proportional vesting encumbrance.
+
+A periods file is a JSON object describing a sequence of unlocking or vesting events,
+with a start time and an array of coins strings and durations relative to the start or previous event.`,
+		Example: `Sample period file contents:
```

Comment:
woaaa, `Example` was always part of the `cobra.Command`? Neat!

Reply:
We should add this to gamm create-pool as well!